packages/slate-tools/package.json

```diff
@@ -56,8 +57,10 @@
     "jest": "22.4.2",
     "minimatch": "^3.0.4",
     "minimist": "1.2.0",
+    "node-ip": "^0.1.2",
```

Comment:
I think this is meant to be `"ip": "^1.1.5",` or the `prompts/confim-private-ip.js` needs to have it's `require()` change.

Reply:
Yes! Got mixed up there. Thanks.